bin/drush-install.sh

```diff
@@ -68,11 +68,11 @@ function set_resource_urls() {
 ##################################
 ## Main
 
-# Get CiviCRm Path and shift to the next option
+# Get CiviCRM Path and shift to the next option
 CIVI_PATH=$1
 shift
 
-if [ "$1" == "--with-sample-data" ]; then
+if [ "$1" == "--with-hrsample-data" ]; then
```

Comment:
well .. currently when you create a new buildkit instance there is no option to not allow it to create the sample data that is shipped with civicrm core .. so I am thinking to add another option to the buildkit called (--with-sample-data) to either allow civicrm to create its sample data or not .. and --with-hrsample-data will be reserved for civihr sample data.